style.css

```diff
@@ -609,8 +609,51 @@ footer {
     display: none;
   }
 
+  .nav-list--item a {
+    text-decoration: none;
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+    color: #3a4a42;
+    padding-left: 20px;
+  }
+
+  .display-menu {
+    list-style: none;
+    width: 101%;
+    height: 105%;
+    padding: 20px;
+    position: absolute;
+    left: 0;
+    top: -10px;
+    background: #fff;
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    border-bottom-right-radius: 30px;
+    border-bottom-left-radius: 30px;
+    box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  }
```

Comment:
- [OPTIONAL] It would be nice if the user cannot scroll up and down while the menu is open so the user cannot leave the menu by accident while its still opened.
 - 
<img width="450" alt="image" src="https://user-images.githubusercontent.com/61919617/230949909-88a6d563-ef02-42cc-b6ff-9d7bb57560a3.png">

Reply:
Thank you for the observation I will make the changes, though optional.